### PR TITLE
add LHCb submodule with conversion to/from LHCb naming scheme

### PR DESCRIPTION
**Pull request by admorris**
Replaces #347

Unlike the previous PR, this diff only touches `src/particle/lhcb` and `tests/lhcb`

The script to create the CSV file is here: [`dump_pdgid_to_lhcb.py`](https://gist.github.com/admorris/977eeec7e50f4a6fde054e6fa2261f18). I'm not sure where to put it.

### CODE REVIEW

File: src/particle/lhcb/__init__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+# Distributed under the 3-clause BSD license, see accompanying file LICENSE
+# or https://github.com/scikit-hep/particle for details.
+
+from __future__ import absolute_import
+
+from typing import Tuple
+
+from .utils import from_lhcb_name, to_lhcb_name
+
+__all__ = (
```
Comment (eduardo-rodrigues) [inline src/particle/lhcb/__init__.py:13]: Given that this is an experiment specific submodule I think it is fine to add `LHCbName2PDGIDBiMap` to the list of "import *".

**Comment by eduardo-rodrigues** [comment]
Thank you for this new "design". I left a few suggestions but otherwise great 👍.

> The script to create the CSV file is here: `dump_pdgid_to_lhcb.py`. I'm not sure where to put it.

A `scripts/` folder in the top-level could be misleading for obvious reasons. I think it is fine (at least as a starting point) to add your script under a top-level `admin/` folder.

**Comment by eduardo-rodrigues** [comment]
Hi @admorris, seems I made a silly manipulation and managed to remove all my review comments and suggestions! Hopefully you have the GitHub emails and can still act upon my comments. Else let me know and I will review again.

**Comment by eduardo-rodrigues** [comment]
> Hi @admorris, seems I made a silly manipulation and managed to remove all my review comments and suggestions! Hopefully you have the GitHub emails and can still act upon my comments. Else let me know and I will review again.

@admorris, any follow-up here? Just too busy, then?

**Comment by admorris** [comment]
Hi @eduardo-rodrigues I have just implemented and pushed the 2 suggestions I see in my inbox. I don't think I received the ones that vanished from GitHub

File: admin/dump_pdgid_to_lhcb.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
```
Comment (eduardo-rodrigues) [inline admin/dump_pdgid_to_lhcb.py:3]: Do write a header to explain what the file is for and what it produces.

Comment (admorris) [reply admin/dump_pdgid_to_lhcb.py:8]: Added a docstring (plus copyright statement) in 38fda43

**Review by eduardo-rodrigues — APPROVED** [review]

**Comment by eduardo-rodrigues** [comment]
Hi @henryiii, this MR has been around for quite a while - mea culpa - and I would like to merge it. I assume that you're OK with this experiment-specific module since you did not comment? I will merge later today.

**Comment by henryiii** [comment]
Yes, I'd be okay with it.

**Comment by eduardo-rodrigues** [comment]
FYI @admorris this is now available on PyPI as version 0.16.3, see https://pypi.org/project/particle/0.16.3/. It will soon appear in Conda ...

As a follow-up we should ensure that this new `lhcb` module is in the documentation.

**Pull request merged**